tools/tpm2_getmanufec.c

```diff
@@ -332,11 +332,13 @@ unsigned char *HashEKPublicKey(void)
         goto hash_out;
     }
 
-    unsigned i;
-    for (i = 0; i < SHA256_DIGEST_LENGTH; i++) {
-        printf("%02X", hash[i]);
+    if (verbose) {
+        unsigned i;
+        for (i = 0; i < SHA256_DIGEST_LENGTH; i++) {
+            printf("%02X", hash[i]);
+        }
+        printf("\n");
     }
```

Comment:
No, because then we end up with additional information that LOG_INFO prints out, filename and line number.... for each printf statement.